readthedocs/proxito/views/hosting.py

```diff
@@ -540,7 +540,7 @@ def _v1(self, project, version, build, filename, url, request):
                     },
                 },
                 "filetreediff": {
-                    "enabled": False,
+                    "enabled": project.addons.filetreediff_enabled,
```

Comment:
FTD doesn't only depend on this value, it depends on the current version being from a PR preview. All those conditions are already checked in 


https://github.com/readthedocs/readthedocs.org/blob/main/readthedocs/proxito/views/hosting.py#L642-L646

Reply:
Yes, I know. However, we want to return if the FTD addons is enabled o not, as we do with all the other addons. The front end validates if it has all the data required to execute the addon or not. We use the `data-validation.js` file there. In this case, if there is no `filetreediff.diff` field, it won't be enabled.